Configuration/PyReleaseValidation/scripts/runTheMatrix.py

```diff
@@ -132,6 +139,12 @@ def runSelected(opt):
                       dest='step1Only',
                       default=False
                       )
+    parser.add_option('--maxSteps',
```

Comment:
@srimanob it looks to me there is no code to consume this option in 8_0_X

Reply:
@fabiocos I removed it as suggestion.